packages/html-reporter/src/testCaseView.tsx

```diff
@@ -68,15 +69,19 @@ function renderAnnotationDescription(description: string) {
   try {
     if (['http:', 'https:'].includes(new URL(description).protocol))
       return <a href={description} target='_blank' rel='noopener noreferrer'>{description}</a>;
-  } catch {}
+  } catch { }
   return description;
 }
 
 function TestCaseAnnotationView({ annotation: { type, description } }: { annotation: TestCaseAnnotation }) {
+  const [hover, setHover] = React.useState(false);
+  const onHover = () => setHover(true);
+  const onLeave = () => setHover(false);
   return (
-    <div className='test-case-annotation'>
-      <span style={{ fontWeight: 'bold' }}>{type}</span>
+    <div className='test-case-annotation' onMouseEnter={onHover} onMouseLeave={onLeave}>
```

Comment:
It looks like you are re-rendering the component that you add listeners to on hover, which might results in unexpected results.

I'd suggest adding a new component next to CopyToClipboard called <CopyContainer><span>aaa</span></CopyContainer> that would allow copying its content. That way we can use it in more places. I would also recommend that the visibility is implemented using CSS.

Reply:
Not fully sure what you mean here . Made the required changes based on my what I could understand from your comment .